README.md

```diff
@@ -73,6 +73,11 @@ Bridge to connect Amazon Alexa to Api.ai using an AWS Lambda Function.
 * Create a new Intent called **Default Bye Event**:
 	* Add **`BYE`** as a trigger Event.
 	* Add or modify any text responses which will be triggered as a goodbye response.
+	* Ensure that all contexts are deleted when this event is triggered.
```

Comment:
_isResponseIncompleted_ requires all contexts are deleted in order to end the session